ArduPlane/quadplane.cpp

```diff
@@ -1869,9 +1869,12 @@ void QuadPlane::control_run(void)
     switch (plane.control_mode->mode_number()) {
     case Mode::Number::QACRO:
         control_qacro();
-        // QACRO uses only the multicopter controller
-        // so skip the Plane attitude control calls below
+        if (!is_tailsitter()) {
```

Comment:
i think you don't need to check for tailsitter here, it is over written anyway, we don't check for taisitter in Qstab ect

Reply:
I think it saves quite a few CPU cycles though...
And is it good design practice to make multiple conflicting assignments to servo outputs within a single iteration of the controller? I'd say it's confusing (in addition to being inefficient) and indicates a need for refactoring.

Reply:
might as well add the check to line 1899 too then. At least its consistent then. As you say would be nice to have a better way to do it.

Reply:
Thanks, I took both your suggestions.
I took out the is_tailsitter check for now, because I can't see how it will interact with the gain interpolation/copter tailsitter/non-copter tailsitter cases. Maybe it will be easier to see how to eliminate the unnecessary calculations once the gain options PR goes in.